internal/handler/handler.go

```diff
@@ -109,6 +109,38 @@ func (handler *Handler) updateDNS(domain *settings.Domain, ip string) error {
 			}
 		}
 	}
+	for _, designated := range domain.DelegatedSubDomains {
```

Comment:
May I know what does the `DelegatedSubDomains` used for?

Reply:
If so, this part of the code is only available for the Google handler?

Reply:
I think no need to check the other providers. I'm not sure if this change will affect the other providers. For example, is the config `DelegatedSubDomains` compulsory? Or for the other providers can just ignore it from the config file?